.github/workflows/build.yml

```
@@ -84,45 +119,91 @@ jobs:
             os: ubuntu-20.04
-            build_opts: --android_abi x86_64 --cmake_extra_defines CMAKE_SYSTEM_NAME=Android CMAKE_SYSTEM_PROCESSOR=x86_64 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib --android
+            build_opts: |-
+              --compile_no_warning_as_error
+              --android
+              --android_abi x86_64
+              --cmake_extra_defines
+              CMAKE_SYSTEM_NAME=Android
+              CMAKE_SYSTEM_PROCESSOR=x86_64
             result_dir: build
             release_config: Release
           - artifact_name: onnxruntime-android-arm64
             os: ubuntu-20.04
-            build_opts: --android_abi arm64-v8a --cmake_extra_defines CMAKE_SYSTEM_NAME=Android CMAKE_SYSTEM_PROCESSOR=aarch64 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib --android
+            build_opts: |-
+              --compile_no_warning_as_error
+              --android
+              --android_abi arm64-v8a
+              --cmake_extra_defines
+              CMAKE_SYSTEM_NAME=Android
+              CMAKE_SYSTEM_PROCESSOR=aarch64
             result_dir: build
             release_config: Release
           - artifact_name: onnxruntime-ios-arm64
             os: macos-12
-            build_opts: --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib --skip_tests --use_xcode --ios --apple_sysroot iphoneos --osx_arch arm64 --apple_deploy_target 16.0
-            build_opts_workaround_protoc: --path_to_protoc_exe /usr/local/opt/protobuf@21/bin/protoc
+            build_opts: |-
+              --compile_no_warning_as_error
+              --skip_tests
+              --use_xcode
+              --ios
+              --apple_sysroot iphoneos
+              --osx_arch arm64
+              --apple_deploy_target 16.0
+              --path_to_protoc_exe /usr/local/opt/protobuf@21/bin/protoc # https://github.com/VOICEVOX/onnxruntime-builder/issues/23#issuecomment-1599594381
             result_dir: build/Release
             release_config: Release-iphoneos
           - artifact_name: onnxruntime-ios-sim-arm64
             os: macos-12
-            build_opts: --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib --skip_tests --use_xcode --ios --apple_sysroot iphonesimulator --osx_arch arm64 --apple_deploy_target 16.0
-            build_opts_workaround_protoc: --path_to_protoc_exe /usr/local/opt/protobuf@21/bin/protoc
+            build_opts: |-
+              --compile_no_warning_as_error
+              --skip_tests
+              --use_xcode
+              --ios
+              --apple_sysroot iphonesimulator
+              --osx_arch arm64
+              --apple_deploy_target 16.0
+              --path_to_protoc_exe /usr/local/opt/protobuf@21/bin/protoc # https://github.com/VOICEVOX/onnxruntime-builder/issues/23#issuecomment-1599594381

```

これがワークアラウンドである、という情報が消えてますね！
でもまあ`protobuf@21`をインストールする方にworkaroundだというコメントはあるので、問題なさそう！

書いても良いかも。書くなら下2つの ios系の方も
```suggestion
              --path_to_protoc_exe /usr/local/opt/protobuf@21/bin/protoc # this is workaround, see https://github.com/VOICEVOX/onnxruntime-builder/issues/23#issuecomment-1599594381

```

---

こうしてみました。これでも既存の方のワークアラウンド表記に結び付くはず…
[`66d3eb3` (#48)](https://github.com/VOICEVOX/onnxruntime-builder/pull/48/commits/66d3eb3484a5dd24364a87f3410ee2508cbc554b)